.changeset/mighty-mangos-complain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/shared': patch
+---
+
+Fix issue with existing directories upon building already built `@clerk/shared`
```

Comment:
This is an issue internal to us, I don't think we'd need to put this into the changelog

Reply:
This issue can be shown in any contributor trying to build the shared package. that's why i have added a changelog entry but i will remove it since you suggest it.